www/_posts/2021-02-02-cordova-fetch-3.0.1.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+author:
+    name: Darryl Pogue
+    url: https://twitter.com/dpogue
+title:  "Cordova Fetch 3.0.1 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that `cordova-fetch@3.0.1` was released in February 2021. This module is responsible for installing platforms and plugins from the npm package registry, and this release adds compatibility for npm version 7.
+
+* [cordova-fetch@3.0.1 on npm](https://www.npmjs.com/package/cordova-fetch)
+
+
+**To upgrade:**
+
+Because `cordova-fetch` is a dependency of the `cordova` command-line tool, you'll need to reinstall the `cordova` package to update:
+
+```bash
+npm uninstall cordova
+npm install cordova
```

Comment:
I believe the docs shows it as a globally installed package, so we might need to reference that here as well?

 ```suggestion
npm uninstall -g cordova
npm install -g cordova
```

Reply:
Installing Cordova globally is one of the biggest sources of problems with permissions and weird bugs 😞 (but yes, you're right, that is what the current docs say)